getting-started/quickstart-guides/angular.md

```diff
@@ -0,0 +1,834 @@
+* [Blurred image placeholder](angular.md#blurred-image-placeholder)
+* [Client-side file uploading](angular.md#uploading-files-in-angular)
+* [Advanced file uploading](angular.md#advanced-file-upload)
+* [Rendering videos](angular.md#rendering-videos)
+
+## **Setup ImageKit Angular SDK**
+
+For this tutorial, it is recommended to create a dummy Angular app, as shown below.&#x20;
+
+**Create a Angular app:**
+
+We will be using the following:
+- Angular 15
+npm uninstall -g @angular/cli
+npm install -g @angular/cli@15
+
+- `node: v14.21.1`
+
+
+Let's use the `ng new <project name>` CLI utility provided by Angular to build a new project:
+
+```bash
+ng new imagekit-angular-app
+```
+
+Navigate to the project directory:
+
+```
+cd imagekit-angular-app/
+```
+
+Open up the project in your text editor of choice, and navigate to `src/app/`. This is where we will do most of our work.
+
+
+&#x20;Install libraries (if not already):
+
+```
+npm install
+```
+
+&#x20;Now run the app:
+
+```
+npm start
+```
+
+In your web browser, navigate to `http://localhost:4200/`
+
+You should see the dummy app created by Angular CLI. 
+
+For simplicity, let's remove everything from `src/app/app.component.html` so we can begin work with a clean slate.
+
+Let's add one line in `src/app/app.component.html` to title our page :
+
+```js
+<h1>Imagekit Angular Demo</h1>
+```
+
+Now we can begin our work.
+
+**Install the ImageKit Angular SDK:**
+
+Installing the ImageKit Angular SDK in our app is pretty simple:
+
+```
+npm install --save imagekitio-angular
+```
+
+#### **Initialize the Angular SDK:**
+
+Before the SDK can be used, let's learn about and obtain the requisite initialization parameters:
+
+* `urlEndpoint` is a required parameter. This can be obtained from the [URL-endpoint section](https://imagekit.io/dashboard/url-endpoints) or the [developer section](https://imagekit.io/dashboard/developer/api-keys) on your ImageKit dashboard.
+* `publicKey` and `authenticationEndpoint` parameters are optional and only needed if you want to use the SDK for client-side file upload. These can be obtained from the [Developer section](https://imagekit.io/dashboard/developer/api-keys) on your ImageKit dashboard.
+
+These details have to be input in the following file.
+
+{% code title="src/app/app.module.ts" %}
+```js
+@NgModule({
+  declarations: [
+    AppComponent
+  ],
+  imports: [
+    BrowserModule,
+    ImagekitioAngularModule.forRoot({
+      publicKey: environment.publicKey,
+      urlEndpoint: environment.urlEndpoint,
+      authenticationEndpoint: environment.authenticationEndpoint
+    })
+  ],
+  providers: [],
+  bootstrap: [AppComponent]
+})
+```
+{% endcode %}
+
+{% hint style="info" %}
+_**Note:**_ _Do not include your _[_API private key_](../../api-reference/api-introduction/api-keys.md#private-key)_ in any client-side code, including this SDK or its initialization. If you pass the `privateKey` parameter while initializing this SDK, it will throw an error._
```

Comment:
Is this true. If not we can remove it.

Reply:
Removing

Reply:
Please don't remove this altogether, rather make it correct. I did update SDK readme to make this correct. You can copy paste from there.

Reply:
Updated